website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,58 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) is enabled, you can declare external endpoints in the [`Destination`](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
+1. You can add the service as a node in the Consul catalog.
+
+#### Register an external service as a Destination
```

Comment:
non-blocking: It would be amazing if you folks could provide some examples. e.g. for `https://www.twitter.com` use this `ServiceDefaults` (I'm guessing with protocol: tcp), and you can then call Twitter from your service with `curl https://www.twitter.com`

Reply:
I think this doc reads a little like a tutorial that ends with a similar "try this" command. I would favor replacing `example.com` with something like `httpbin.org` with a simple response. WDYT @trujillo-adam @lkysow?

Reply:
Ahh I see, sorry I didn't look at the full context. I still think that reading the table of caveats is hard to understand when mapping to what my use-case is. I don't have strong feelings about replacing httpbin but I do think that having examples for when I should use L4 vs L7 and what the servicedefaults and eventual `curl` calls would look like would make it much easier to understand.

Reply:
I agree the table is challenging, but I struggled to find a better representation. The best solution would be to have some permissive or auto setting to make figuring this out more accessible, but that's forward-looking. Open to any suggestions here.

Reply:
W/r/t examples, I'm in favor of using as realistic configurations as possible so that readers can form a mental model of what we are describing. Regarding the table of behaviors, I actually think this version of the table is pretty straightforward. There are a few blank cells under the Notes column that we should complete, but as an outside reader I feel that I would be able to align my use case to the information being presented. Maybe Notes should be changed to Description.

Reply:
OK, I think we'll keep it as-is for now and work on more examples as this feature develops. We have some additional features planned where we can add to this.